File: C++/Algorithms/Sorting/BubbleSort.cpp
#include<iostream>
using namespace std;


int main(){

   int n;
   cout<<"Enter size of Array : ";
   cin>>n;
   int arr[n];
   cout<<"Enter the array : ";
   for(int i=0;i<n;i++){
       cin>>arr[i];
   }
   //Bubble Sort Algorithm
   for(int i=0;i<n;i++){
       for(int j=0;j<n-i-1;j++){
           if(arr[j]>arr[j+1]){
               int temp = arr[j];
               arr[j]   = arr[j+1];
               arr[j+1]   = temp;
           }
       }
   }
   for(int i=0;i<n;i++){
       cout<<arr[i]<<" ";
   }

   return 0;
}

File: Python/Algorithms/Sorting/Bubble_sort.py
# Python program for Bubble Sort

number = int(input("Please Enter the Total Number of Elements : ")) # number yang perlu idiisi
a = [int(input("Please enter the %d Element of List1 : " %i)) for i in range(number)]
# masukkan data sesuai elemen list1

i = 0 # lakukan perulangan while
while(i < number -1):
    j = 0
    while(j < number - i - 1):
        if(a[j] > a[j + 1]):
             temp = a[j]
             a[j] = a[j + 1]
             a[j + 1] = temp
        j = j + 1
    i = i + 1

print("The Sorted List in Ascending Order : ", a)
